### DISCUSSION TITLE
Have a tooltip which will have same option feature as toolbar.

### DISCUSSION THREAD

**Discussion by pinak3748** [facebook/lexical#4679] [Ideas]
The suggested feature is to incorporate an enhanced tooltip in a rich text editor, providing users with quick access to essential formatting options similar to those found in the toolbar. This tooltip would appear when the user hovers over a selected text or within the editor itself.

Elaboration: The enhanced tooltip would contain a condensed set of commonly used formatting options, such as bold, italic, underline, font size, font color, and alignment. By including these options directly in the tooltip, users can quickly modify the selected text without needing to navigate to the toolbar. This feature saves time and enhances the editing experience by providing a convenient and intuitive way to apply formatting while working with text.

![image](https://github.com/facebook/lexical/assets/59458280/9007646f-77e5-4523-a059-a8294fb1413c)

**Comment by nandorojo** [comment]
https://github.com/facebook/lexical/blob/0775ab929e65723433626fa8c25900941e7f232f/packages/lexical-playground/src/plugins/TextFormatFloatingToolbarPlugin/index.tsx

I think that's what you want.

**Reply by SalahAdDin** [reply]
@pinak3748 how did you handle it?
